src/components/RequestModal/AdvancedRequester/index.tsx

```diff
@@ -21,12 +21,15 @@ const messages = defineMessages({
   loadingprofiles: 'Loading profiles…',
   loadingfolders: 'Loading folders…',
   requestas: 'Request As',
+  languageprofile: 'Language Profile',
+  loadinglanguages: 'Loading folders…',
```

Comment:
```suggestion
  loadinglanguages: 'Loading language profiles…',
```

Reply:
Or, we can just use the `loadingProfiles` string that already exists.  I think we should either use the same string for both, or change `loadingProfiles` to `'Loading quality profiles…'`